index.da.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Et åbent brev for at fjerne Richard M. Stallman fra alle lederskabspositioner
+description: Vi, de underskrevende, vil have at hele FSF lederskabet at træde af og for Richard M. Stallman at blive fjernet fra alle lederskabspositioner. Vi vil ikke længere acceptere hans opførsel, lederskabsrolle og på anden vis holde hans skadefulde og farlige ideologi som acceptabelt.
+image: /assets/og-image.png
+twitter:
+    card: summary_large_image
+share_text: Jeg støtter dette kald for at hele FSF-bestyrelsen skal træde af og for at Richard M. Stallman skal fjernes fra alle lederskabspositioner.
+---
+
```

Comment:
Per https://github.com/rms-open-letter/rms-open-letter.github.io/pull/2545#issuecomment-810722881 the share links won't be generated without this fix :)